draft-privacypass-rate-limit-tokens.md

```diff
@@ -521,10 +521,10 @@ struct {
    uint16_t token_type = 0x0003;
    uint8_t token_key_id;
    uint8_t blinded_msg[Nk];
-   uint8_t request_key[32];
+   uint8_t request_key[97];
    uint8_t name_key_id[32];
    uint8_t encrypted_origin_name<1..2^16-1>;
-   uint8_t request_signature[64];
+   uint8_t request_signature<1..2^16-1>;
```

Comment:
This is because ASN.1 encoded ECDSA signatures are variable length, annoyingly

Reply:
I'd really prefer we revert this change, provided we have an encoding that produces fixed-length signatures.

Reply:
We can move to r || s encoding

Reply:
Done.